src/components/learner-credit-management/data/utils.js

```diff
@@ -680,36 +682,50 @@ export const getAssignableCourseRuns = ({ courseRuns, subsidyExpirationDatetime,
     enrollStart: courseRun.hasEnrollStart ? dayjs.unix(courseRun.enrollStart).toISOString() : null,
     upgradeDeadline: dayjs.unix(courseRun.upgradeDeadline).toISOString(),
   }));
+
   const assignableCourseRunsFilter = ({
-    enrollBy, enrollStart, start, isActive, hasEnrollBy, isLateEnrollmentEligible,
+    enrollBy, enrollStart, start, hasEnrollBy, hasEnrollStart, isActive, isLateEnrollmentEligible,
   }) => {
+    const isWithinEnrollByDateThreshold = isEnrollByDateWithinThreshold({
+      hasEnrollBy,
+      enrollBy,
+      isLateRedemptionAllowed,
+    });
+    const isWithinStartDateThreshold = isStartDateWithinThreshold({
```

Comment:
nit: perhaps could be simplified as `isEnrollByDateValid` and `isStartDateValid`?